tasks/classes/classes.js

```diff
@@ -11,15 +11,33 @@
  * 1) property grade
  * 2) constructor that pass grade to the object
  * 3) method getGrade that returns property grade
- * 4) static method compareGrades that pass two instances of Student and compare their grades,
- *    1 if first greater
- *    0 if equal
- *    -1 if second greater
  */
 //put your code here
 
-const person = //create instance of class person
-const student = //create instance of class student
+
+class Person {
+    constructor(firstName, lastName) {
+      this.firstName = firstName;
+      this.lastName = lastName;
+    }
+    getFullName() {
+      return `${person.firstName} ${person.lastName}`
+    }
+    
+  };
+  const person = new Person();
+
+class Student extends Person {
+    constructor(firstName, lastName, getFullName, grade) {
```

Comment:
Yo do not need to put methods (getFullName) inside the constructor. They will be accessed in the inherited classes anyway.